MODULE.bazel

```diff
@@ -1,21 +1,38 @@
 "Bazel dependencies"
 
 module(
-    name = "com_myorg_rules_mylang",
+    name = "rules_apko",
     version = "0.0.0",
     compatibility_level = 1,
 )
 
 bazel_dep(name = "bazel_skylib", version = "1.4.1")
 bazel_dep(name = "platforms", version = "0.0.5")
+bazel_dep(name = "aspect_bazel_lib", version = "1.34.5")
 
 bazel_dep(name = "gazelle", version = "0.29.0", dev_dependency = True, repo_name = "bazel_gazelle")
 bazel_dep(name = "bazel_skylib_gazelle_plugin", version = "1.4.1", dev_dependency = True)
-bazel_dep(name = "aspect_bazel_lib", version = "1.32.1", dev_dependency = True)
 bazel_dep(name = "buildifier_prebuilt", version = "6.1.0", dev_dependency = True)
 
-mylang = use_extension("//mylang:extensions.bzl", "mylang")
-mylang.toolchain(mylang_version = "1.14.2")
-use_repo(mylang, "mylang_toolchains")
+toolchain = use_extension("//apko:extensions.bzl", "apko")
+toolchain.toolchain(apko_version = "v0.0.0")
+use_repo(toolchain, "apko_toolchains")
 
-register_toolchains("@mylang_toolchains//:all")
+register_toolchains("@apko_toolchains//:all")
+
+# locks
+lock = use_extension(
```

Comment:
nit: maybe `apk` instead of `lock` so it's `apk.translate_lock` below, more 1:1 with `npm.translate_lock`

Reply:
agreed